goalie/options.py

```diff
@@ -167,11 +167,9 @@ def export(self, metric):
             "hausdorff_number",
             "gradation_factor",
         )
-        metric_parameters = {
-            "dm_plex_metric": {key: self[key] for key in petsc_specific}
-        }
-        metric_parameters["num_iterations"] = self["num_parmmg_iterations"]
-        metric.set_parameters(metric_parameters)
+        mp = {"dm_plex_metric": {key: self[key] for key in petsc_specific}}
+        mp["dm_plex_metric_num_iterations"] = self["num_parmmg_iterations"]
+        metric.set_parameters(mp)
```

Comment:
@jwallwork23 - I am curious why you are reverting to using `mp` instead of keeping the recently updated `metric_parameters`  for improved readability.

Reply:
Fair point. It was to avoid line continuation but readability is important. Will propose an alternative.

Reply:
Okay how's 758c4a6a23c5d3062f4f432fda90be08155a77f2?